examples/notebooks/TextCNN.ipynb

```diff
@@ -983,7 +983,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
No need to change this. You might have edited and saved the notebook in a local Jupyter notebook instance. Did you?
```suggestion
   "display_name": "Python 3",
```

Reply:
Yes 😅 I will try to get these unnecessary changes out.